sdk/core/core-auth/src/azureKeyCredential.ts

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
 // Copyright (c) Microsoft Corporation.
```

Comment:
duplicate

Reply:
I like the idea of making the rule check for comment lines that just start with "// Copyright" and "// Licensed"  at the top of the file and removing them in the fix pass.